app/src/components/Order.js

```diff
@@ -3,16 +3,24 @@ import {withStyles} from '@material-ui/core/styles';
 import classNames from 'classnames';
 
 import {Styles} from '../styles/Order';
+import {Typography, Box, Tooltip} from '@material-ui/core';
 
 class Order extends Component {
 
     render() {
-        const {classes, type, quantity} = this.props;
+        const {classes, type, quantity, maxQuantitySum} = this.props;
+        const quantityBoxSize = (quantity / maxQuantitySum) * 100;
 
         return (
-            <span className={classNames(classes.rectangle, type === 'bid' ? classes.bid : classes.ask)}>
-                {quantity}
-            </span>
+            <Box className={classNames(classes.rectangle, type === 'bid' ? classes.bid : classes.ask)} style={{minWidth:`${quantityBoxSize}%`, maxWidth: `${quantityBoxSize}%`}}>
+                {quantityBoxSize > 1.5 ?
+                    (
+                        <Typography className={classes.text}>{quantity}</Typography>
+                    ) : (
+                        <Tooltip title={quantity} placement='cursor'><span style={{width: '100%', height: '100%'}}/></Tooltip>
```

Comment:
can you put the width and height the stylesheet ?, would probably need to use `className`